core/src/test/java/org/testcontainers/images/RemoteDockerImageTest.java

```diff
@@ -0,0 +1,32 @@
+package org.testcontainers.images;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.not;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.when;
+
+import org.junit.Test;
+import org.mockito.Mockito;
+import org.testcontainers.utility.Base58;
+
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+
+public class RemoteDockerImageTest {
+
+    @Test
+    public void toStringContainsOnlyImageName() {
+        String imageName = Base58.randomString(8).toLowerCase();
+        RemoteDockerImage remoteDockerImage = new RemoteDockerImage(imageName);
+        assertThat(remoteDockerImage.toString(), containsString(imageName));
+        assertThat(remoteDockerImage.toString(), not(containsString("imageNameFuture")));
+    }
+
+    @Test
+    public void toStringWithExceptionContainsOnlyImageNameFuture() throws Exception {
+        Future<String> imageNameFuture = Mockito.mock(Future.class);
```

Comment:
No need to Mockito, can be as simple as:
```java
CompletableFuture<String> future = new CompletableFuture<>();
        future.completeExceptionally(new RuntimeException("arbitrary"));
```

Reply:
Done in https://github.com/testcontainers/testcontainers-java/pull/2450/commits/1af3325d90bddb46e8a66fe1494639c550bdc574